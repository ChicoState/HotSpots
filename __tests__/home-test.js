import 'react-native';
import React from 'react';
import DealScreen from '../src/screens/DealsScreen';
import renderer from 'react-test-renderer';


test('deals snapshot', () => {
    const snap = renderer.create(<DealScreen />).toJSON;
    expect(snap).toMatchSnapshot();

});
