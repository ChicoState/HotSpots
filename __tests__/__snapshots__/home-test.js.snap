// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`deals snapshot 1`] = `[Function]`;
